README.md

```diff
@@ -117,28 +117,16 @@ archive { '/tmp/test100k.db':
 
 ### Puppet URL
 
-Below is an example of how to deploy a tar.gz to a local directory when it is
-served via the ```puppet:///``` style url which is not currently supported.
+Since march 2017, the Archive type also supports puppet url's. Here is an example
```

Comment:
@hajee Actually, you might be ok.  Ruby isn't the default provider, *but* it is still the parent provider for `curl` and `wget` providers which override the parent's `download` method.  Could you double-check?  For example, looks like things might still go wonky if a user tries to specify a `checksum_url`.